bucket/covergroup.py

```diff
@@ -22,17 +22,18 @@ class Covergroup(CoverBase):
     """This class groups coverpoints together, and adds them to the hierarchy"""
 
     @validate_call
-    def __init__(self, name: str, description: str):
+    def __init__(self, name: str | None = None, description: str | None = None):
         """
         Parameters:
             name: Name of covergroup
             description: Description of covergroup
         """
 
-        self.name = name
-        self.description = description
+        self.name = name if name is not None else self.NAME
```

Comment:
If self.name ends up as None this should be an error right? Or perhaps could default to the name of the class

At the moment it will error below at `self.full_path = self.name.lower()` with something not user friendly

Reply:
Isn't this still the case?